lib/rabbit-topology.js

```diff
@@ -49,7 +62,9 @@ module.exports = function rabbitTopology() {
 
     Model[methodName] = function queueProducer(params) {
       debug(`${modelName}.${methodName}(%o)`, params)
-      QueueModel.rabbit.exchange.publish(params, { key: queue })
+      if(QueueModel.rabbit && QueueModel.rabbit.exchange) {
+        QueueModel.rabbit.exchange.publish(params, { key: queue })
+      }
```

Comment:
Lets add an else condition and add a debug message that says that the queue is not yet initialised in case this method is called prior to the queue initialisation.